config/root_config.go

```diff
@@ -358,3 +360,29 @@ func publishMapping(sc exporter.MetadataServiceExporter) error {
 	}
 	return nil
 }
+
+// Process receive changing listener's event, dynamic update config
+func (rc *RootConfig) Process(event *config_center.ConfigChangeEvent) {
+	logger.Info("CenterConfig process event:\n%+v", event)
```

Comment:
logger.Debugf

Reply:
I think change to logger.Infof is better

Reply:
done